Corona-Warn-App/src/main/res/layout/fragment_submission_done_no_consent.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/submission_done_no_consent_container"
+        android:contentDescription="@string/submission_done_no_consent_title"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true"
+        tools:context=".ui.submission.fragment.SubmissionDoneNoConsentFragment">
+
+        <include
+            android:id="@+id/submission_done_no_consent_header"
+            layout="@layout/include_header"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="wrap_content"
+            app:icon="@{@drawable/ic_close}"
+            app:title="@{@string/submission_done_no_consent_title}"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <include
+            android:id="@+id/submission_done_no_consent_content"
+            layout="@layout/include_submission_done_no_consent"
```

Comment:
Do we really need a separate layout file for the contents of this include? Is it meant to be reusable? I'm not seeing it used anywhere else. It would be nice to just put the contents of this layout directly in the fragment to reduce the amount of includes that we're using. But if it's meant to be reused some place else, then I can understand.

Reply:
Agreed, the submission with consent content is different, so it is not required to be re-usable. You could just add the contents in this layout to simplify things a bit.

Reply:
I think it's fine, just a matter of preferences. I would leave it as it is.

Reply:
I think it's important to not saturate one layout file too much. hence I've gone the way of using layouts.

@AlexanderAlferov @chiljamgossow  what do you think?